xml/System.Text.RegularExpressions/CaptureCollection.xml

```diff
@@ -784,10 +841,19 @@
         <Parameter Name="index" Type="System.Int32" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1;netcore-3.1;net-5.0" />
       </Parameters>
       <Docs>
-        <param name="index">To be added.</param>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <param name="index">The zero-based index of the element to get or set.</param>
+        <summary>Gets or sets the element at the specified index.</summary>
```

Comment:
Wouldn't set throw an exception here?

Reply:
Fixed. Do we document exceptions here too for the set method since it's read-only?